packages/frontend/src/features/semanticViewer/runners/SemanticViewerResultsRunner.ts

```diff
@@ -239,3 +262,27 @@ export class SemanticViewerResultsRunner implements IResultsRunner {
         return this.rows;
     }
 }
+
+export class SemanticViewerResultsRunnerFrontend extends BaseResultsRunner {
```

Comment:
So in the frontend we use react-query etc. to run the pivot. in the BE this might be different